### PR TITLE
Youtube tag fix settings

### LINKED ISSUE #15: Youtube tag configuration has no effect
Hi, the settings `YOUTUBE_THUMB_ONLY` and `YOUTUBE_THUMB_SIZE` of the Youtube tag have no effect on the behavior of the plugin, in fact they have not been declared in the file mdx_liquid_tags as for the other plugins.
In the documentation it is noted that these parameters must be added to the Pelican configuration but not in the form "Out-of-tree tag".

### PR DESCRIPTION
**Pull request by ysard**
Hi, this PR should fix #15 and partially solve #7 (but yes there is a duplication of the documentation in the project :s).

- Settings are added in the "in-tree form";
- doc is updated
- I added the ability to open the link in the Youtube thumbnail mode into a new tab
`rel` tag attributes are added according to the vulnerability described [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy).

Thanks for reading.

### CODE REVIEW

**Comment by mirekdlugosz** [comment]
hi @ysard , as expected, after merging #14 there are now conflicts here. Could you please rebase on top of `main` and force push?

**Comment by ysard** [comment]
Hi, thanks for accepting the 2 PR, here is the push force ;)

**Review by mirekdlugosz — APPROVED** [review]
Thanks for another good PR :) .

I see there some troubles with CI again. I have a some time now, so I'm going to merge, try to fix CI and attempt release.

**Comment by ysard** [comment]
Hum, indeed I'm sorry, I forgot to edit the test to reflect my addition to the thumbnail link.
https://github.com/pelican-plugins/liquid-tags/blob/141e7138bdcf6acf4ed93660cd9a5406d6ccf486/pelican/plugins/liquid_tags/test_youtube.py#L42

Thanks for the edit.

**Comment by justinmayer** [comment]
@mirekdlugosz said:

> Could you please rebase on top of `main` and force push to your branch?

As of today, this shouldn't be necessary any longer, as GitHub added the [ability for maintainers to do this directly](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-suggestions-to-update-pull-request-branches).

I just enabled that for this repository, so keep an eye out for the **Update Branch** button, tap the down-arrow next to it, choose the _Update with rebase_ sub-menu option, and then tap the renamed **Rebase Branch** button. 🎉

**Pull request merged**
